lib/fbe/iterate.rb

```diff
@@ -214,6 +219,12 @@ def over(timeout: 2 * 60, &)
         [
           repo,
           @fb.query(
+            "(agg (and
+              (eq what 'iterate')
+              (eq where 'github')
+              (eq repository #{repo}))
+            (first #{@label.tr('-', '_')}))"
```

Comment:
@Yegorov don't do this, let's simply raise exception if the formation is wrong

Reply:
@yegor256 Then we won't be able to read old values ​​from separate marker facts. Old markers will remain in factbase, this is what we need?

Reply:
@Yegorov yes, it's fine, we'll just delete them (I have scripts for that)

Reply:
@yegor256 fixed!